src/routes/conversation/[id]/+server.ts

```diff
@@ -379,6 +379,8 @@ export async function POST({ request, locals, params, getClientAddress }) {
 					webSearch: webSearch ?? false,
 					toolsPreference: toolsPreferences ?? {},
 					promptedAt,
+					ip: getClientAddress(),
+					userName: locals.user?.username,
```

Comment:
```suggestion
					username: locals.user?.username,
```

nit to be consistent

Reply:
handled in https://github.com/huggingface/chat-ui/pull/1167/commits/f98f669e780c6d2f12eadc8bddf99baebf267fcf